### PR TITLE
Fixes #11226: Allowed network 0.0.0.0/0 is not currently supported by Apache

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/11226

### CODE REVIEW

File: techniques/system/server-roles/1.0/network-check.st [hunk 1]
```diff
@@ -41,6 +45,14 @@ bundle agent root_networks_check
 
       "index"             slist => getindices("network_file");
 
+    has_all_granted::
+      "network_acl[nodes_22]"  string => "Allow from all${const.n}${nodes_generate_22}";
```
Comment (amousset) [inline techniques/system/server-roles/1.0/network-check.st]: Why adding other allow directives if we allow from all?

**Comment by peckpeck** [comment]
Commit modified

File: techniques/system/server-roles/1.0/network-check.st [hunk 2]
```diff
@@ -41,6 +45,14 @@ bundle agent root_networks_check
 
       "index"             slist => getindices("network_file");
 
+    has_all_granted::
+      "network_acl[nodes_22]"  string => "Allow from all$";
```
Comment (amousset) [inline techniques/system/server-roles/1.0/network-check.st]: no $

**Comment by peckpeck** [comment]
Commit modified

**Pull request merged**